package.json

```diff
@@ -142,6 +143,7 @@
     "eslint-plugin-react-hooks": "1.7.0",
     "eslint-plugin-test-names": "1.0.1",
     "esprima": "4.0.1",
+    "execa": "4.0.0",
```

Comment:
somehow the stdout of the "normal" exec from `child_process` was giving inconsistent and incomplete results. i did not investigate as to why, but fell back on this lib for now.